packages/dso-toolkit/components/Componenten/tile/tile.config.yml

```diff
@@ -0,0 +1,6 @@
+notes: |
+  Breedte van het element wordt gedicteerd door de bootstrap class van het omliggende element.
```

Comment:
`bootstrap class` -> zouden we dit anders moeten noemen? "Bootstrap grid classes"?

In de documentatiepagina van lay-out noemen we het:

> Voor de horizontale verdeling wordt de 12 koloms structuur van het Bootstrap 3 grid gebruikt. Met dit systeem kunnen onderdelen op de pagina worden gepositioneerd en kan worden geprogrammeerd wat het responsive gedrag van deze positionering is.

Reply:
Of gewoon algemeen: "Breedte van het element wordt gedicteerd door het omliggende element."

Reply:
(Dit dan natuurlijk ook aanpassen bij whitebox).